test/ut/nas/test_import_nodep.py

```diff
@@ -0,0 +1,42 @@
+"""To test the cases of importing NAS without certain DL libraries installed."""
+
+import argparse
+import subprocess
+import sys
+
+import pytest
+
+
+def import_related(mask_out):
+    import nni
+    nni.set_default_framework(mask_out)
```

Comment:
should we prevent the user from calling this function repeatedly?

Reply:
We don't. In case they change the framework in the middle of a script, modules already imported will keep the same. Newly imported modules will respect the new setting.